Frameworks/UIKit/UIApplicationMain.mm

```diff
@@ -336,7 +340,7 @@ void _UIApplicationShutdown() {
     THROW_NS_IF_FAILED(comPtr.As(&map));
 
     NSMutableDictionary* userInput = nil;
-    THROW_NS_IF_FAILED(Collections::WRLToNSCollection(map, &userInput));
+    THROW_NS_IF_FAILED(::Collections::WRLToNSCollection(map, &userInput));
```

Comment:
Yes, it's to disambiguate between global Collections and Windows::Foundation::Collections.

---
In reply to: [112285489](https://github.com/Microsoft/WinObjC/pull/2569#discussion_r112285489) [](ancestors = 112285489)